spot-contracts/contracts/FeePolicy.sol

```diff
@@ -307,23 +299,55 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     /// @inheritdoc IFeePolicy
     /// @dev Swapping by minting perps reduces system dr, i.e) drPost < drPre.
     function computeUnderlyingToPerpVaultSwapFeePerc(
-        uint256 /*drPre*/,
+        uint256 drPre,
         uint256 drPost
     ) external view override returns (uint256) {
         // When the after op deviation ratio is below the bound,
         // swapping is disabled. (fees are set to 100%)
-        return (drPost < deviationRatioBoundLower ? ONE : vaultUnderlyingToPerpSwapFeePerc);
+        if (drPost <= drHardBounds.lower) {
+            return ONE;
+        }
+        return
+            LineHelpers
+                .computePiecewiseAvgY(
+                    Line({ x1: drHardBounds.lower, y1: flashMintFeePercs.upper, x2: ONE, y2: flashMintFeePercs.lower }),
+                    Line({ x1: ONE, y1: flashMintFeePercs.lower, x2: drHardBounds.upper, y2: flashMintFeePercs.lower }),
```

Comment:
At first read, I thought the second `flashMintFeePercs.lower` should have been upper. But as-is means the line is flat on the other side of DR.

Looks coorect, but just verifying.

Reply:
Right. Now the breakpoint is dr = 1. In the next PR, we'll add `drSoftBounds` (similar to billy) which will allow us to configure the breakpoint ..